component/Header.tsx

```diff
@@ -1,15 +1,15 @@
-import getConfig from 'next/config'
 import Image from 'next/image'
-import { useRouter } from 'next/router'
 import React from 'react'
+import getConfig from 'next/config'
+import { useRouter } from 'next/router'
 
 const { publicRuntimeConfig } = getConfig()
 
-const Header = () => {
+const Header = ({fixedPosition = true}) => {
     const toDownloadLink = () => window.open(publicRuntimeConfig.DOWNLOAD_BETTERSOCIAL_APP_URL, '_blank')
     
     return (
-        <div className="flex w-full md:max-w-M lg:max-w-M xl:max-w-M justify-between items-center min-h-[45px] fixed top-0 p-4">
+        <div className={`flex w-full md:max-w-M lg:max-w-M xl:max-w-M justify-between items-center min-h-[45px] top-0 p-4 ${fixedPosition ? 'fixed' : ''}`}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `fixedPosition` is a good addition for controlling the positioning of the Header component. However, it's not clear what type this prop is supposed to be. It seems like it should be a boolean based on its usage in the ternary operator, but it would be better to explicitly define the prop types for clarity and to prevent potential misuse.

```diff
- const Header = ({fixedPosition = true}) => {
+ const Header = ({fixedPosition = true}: {fixedPosition?: boolean}) => {
```
This change will make it clear that `fixedPosition` is expected to be a boolean value.


<!-- This is an auto-generated comment by OSS CodeRabbit -->